runtime/tests/tagged_ptr.c

```diff
@@ -45,7 +45,7 @@ void testTaggedToInt() {
     // int on stack
     for(int64_t i=0; i <= NTESTS; i ++) {
         int64_t val = rand();
-        TaggedPtr ptr = bitsToTaggedPtr(val);
+        TaggedPtr ptr = bitsToTaggedPtr(val | IMMEDIATE_FLAG);
         int64_t prtVal = taggedToInt(ptr); //? work only with positive values
         assert(prtVal == val);
```

Comment:
I don't think this test is correct, `rand()` doesn't cover full `int64_t` range. We also need to consider intimidate int range when creating the int.

Reply:
I'll correct it.